### DISCUSSION TITLE
Required words: AND/OR

### DISCUSSION THREAD

**Discussion by LethalCorpse** [pymedusa/Medusa#11868] [Ideas & Feature requests]
Say you've got a pair of shows consisting of a UK original and a US remake. Medusa has a very hard time telling the difference, and will frequently grab episodes from one as episodes from the other. We can try working around this by adding, for example "US" and "2021" to the required words list for the show. This almost works, in that it should only grab releases containing at least one of those words. There's a problem if you also have a global list requiring, say, x265.

Unfortunately, the show list is just appended to global list to make an effective list for that show, with an OR condition, not an AND condition. So Medusa will then grab any release which contains "US" OR "2021" OR "x265". This breaks the functionality of both of the lists - you'll get x265 releases of the wrong show, or x264 releases of the right show.

For this to work properly there needs to be control over whether each required word is AND (in addition to), or "OR" (instead of) other required words.

**Comment by nydave69** [comment]
I see this problem all the time so I......

require for a match that the Scene Exception for the show is
Angel (US) (1999) [eng] {imdb-tt0162065} {tmdb-2426}
as an example.
I did this so it would stop messing up shows that were complete,  note this only works if the show exceptions don't contain anything like  'Angel' that was loaded by the system and can't be removed.

**Comment by KagamiKuno** [comment]
I do a little change on my pymedusa, changing from "OR" to "AND" behavior, tested and now is working as intendend, gonna open a pr to see if they want to push the change.
